node/src/accountant/big_int_db_processor.rs

```diff
@@ -1741,16 +1785,17 @@ mod tests {
         let database_value_1: i64 = 12222;
         let database_value_2: i64 = 23333444;
         let database_value_3: i64 = 5555;
+        let slope: i64 = -3;
```

Comment:
This slope would be more intention-revealing if it had a realistic value--a big one, perhaps.